src/wp-includes/class-wp-text-diff-renderer-table.php

```diff
@@ -30,6 +30,30 @@ class WP_Text_Diff_Renderer_Table extends Text_Diff_Renderer {
 	 */
 	public $_trailing_context_lines = 10000;
 
+	/**
+	 * Title of the item being compared.
+	 *
+	 * @since 6.4.0 Explicitly declare dynamic class property to ensure PHP 8.2 compatibility.
```

Comment:
The `@since` explanation is not required, though it can help to share why it was added.

What about the explanation itself? 
I think it can safely be shortened to remove the PHP part. Why? With the changes made to magic methods, all non-declared properties raise a deprecation (when in `WP_DEBUG` is on). It's not limited to PHP 8.2 and newer. Thus, explaining it is for PHP 8.2 compatibility will likely cause confusion in the future.

```suggestion
	 * @since 6.4.0 Declared a previously dynamic property.
```